server/region_syncer/client.go

```diff
@@ -186,8 +186,10 @@ func (s *RegionSyncer) StartSyncWithLeader(addr string) {
 				}
 				stats := resp.GetRegionStats()
 				regions := resp.GetRegions()
+				buckets := resp.GetBuckets()
 				regionLeaders := resp.GetRegionLeaders()
 				hasStats := len(stats) == len(regions)
+				hasBuckets := len(buckets) == len(regions)
```

Comment:
Is it possible to have more than one bucket in a region?

Reply:
It is impossible,  one region has only one buckets properties include many buckets.